### DISCUSSION TITLE
Colab for this?

### DISCUSSION THREAD

**Discussion by quintendewilde** [volotat/SD-CN-Animation#97] [Q&A] [answered]
Hi is there a Google Colab for this?

Or is this able to run on a Mac M1 (probably not)

**Comment by conor-is-my-name** [comment]
Google colab is blocking most stable diffusion projects now

**Comment by quintendewilde** [comment]
Is there something similar that exist. So I do not have to run it local? 😅

Op ma 15 mei 2023 om 06:11 schreef cbolich ***@***.***>

> Google colab is blocking most stable diffusion projects now
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/volotat/SD-CN-Animation/discussions/97#discussioncomment-5900594>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOARUOVW726GQ4DDGWPNF3XGGUGPANCNFSM6AAAAAAYBKHQZA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

      www.dewildequinten.be

**Reply by conor-is-my-name** [reply]
you can rent a cloud instance.  Vast.ai seems to have good prices, but I have not personally used it myself.

**Comment by quintendewilde** [comment]
@cbolich Thanks for the reply. Any idea what kind of hardware would suffice for this?
2xV100 Tesla? If you have an idea of a sufficient setup, could you share?

**Reply by conor-is-my-name** [reply, marked as answer]
I run it locally on a 3090, start with one of those and see how it goes.  I have no experience with multiple GPU systems, but from what I know you are probably better off working with a single GPU for this package
